src/content/operations/nat-traversal/_index.en.md

```diff
@@ -0,0 +1,415 @@
+all ports, and make sure that the gateways have such port open, the gateways behind
+the on-premises routers which have a single external IP with no IP routing or mapping to the
+private network should have a dedicated and distinct port number within his network for
+the NATT discovery protocol, as well as for encapsulation protocol.
+
+```bash
+kubectl label node $GWA --kubeconfig A gateway.submariner.io/natt-discovery-port=4491
+kubectl label node $GWA --kubeconfig A gateway.submariner.io/udp-port=4501
+kubectl label node $GWB --kubeconfig B gateway.submariner.io/natt-discovery-port=4492
+kubectl label node $GWB --kubeconfig B gateway.submariner.io/udp-port=4502
+kubectl label node $GWC --kubeconfig C gateway.submariner.io/natt-discovery-port=4491
+kubectl label node $GWC --kubeconfig C gateway.submariner.io/udp-port=4501
+kubectl label node $GWD --kubeconfig D gateway.submariner.io/natt-discovery-port=4492
+kubectl label node $GWD --kubeconfig D gateway.submariner.io/udp-port=4502
+
+# restart the gateways to pick up the new setting
+for cluster in A B C D;
+do
+  kubectl delete pod -n submariner-operator -l app=submariner-gateway --kubeconfig $cluster
+done
+```
+
+{{% notice warning %}}
+If **HA** is configured on the on-premises clusters, **each gateway** behind the routers
+**should have a dedicated UDP port**. For example if we had 2 clusters and 2 gateways on each network,
+4 ports would be necessary.
+{{% /notice %}}
+
+#### Router port mapping for multiple on-premises
+
+Under this configuration it's important to map the UDP ports on the 2.1.1.1 router to the private
+IPs of the gateways.
+
+##### On the 2.1.1.1 router
+
+| External IP | Port | Internal IP   | Port | Protocol |
+|:------------|:-----|:--------------|:-----|:---------|
+| 2.1.1.1     | 4501 | 192.168.0.4   | 4501 | UDP      |
+| 2.1.1.1     | 4491 | 192.168.0.4   | 4491 | UDP      |
+| 2.1.1.1     | 4502 | 192.168.128.4 | 4502 | UDP      |
+| 2.1.1.1     | 4492 | 192.168.128.4 | 4492 | UDP      |
+
+##### On the 1.1.1.1 router
+
+| External IP | Port | Internal IP   | Port | Protocol |
+|:------------|:-----|:--------------|:-----|:---------|
+| 1.1.1.1     | 4501 | 10.0.0.1      | 4501 | UDP      |
+| 1.1.1.1     | 4491 | 10.0.0.1      | 4491 | UDP      |
+| 1.1.1.1     | 4502 | 10.1.0.1      | 4502 | UDP      |
+| 1.1.1.1     | 4492 | 10.1.0.1      | 4492 | UDP      |
+
+{{% notice note %}}
+Without port mapping it's entirely possible that the connectivity will be established without
+issues, this happens due to the fact that route's NAT will not generally modify the src port
+of the outgoing UDP packets, and future packets arriving this port will be redirected to the
+internal IP which initiated connectivity, but if the 2.1.1.1 router randomizes the source port on
+NAT, or if other applications on the internal network were already using the 4501-4502,4491-4492
+ports, the remote ends would not be able to contact gateway C or D over the expected ports.
```

Comment:
Similar changes as with the section starting on L266.

Reply:
done.